Workbooks/PaloAltoOverview.json

```diff
@@ -1,1509 +1,1509 @@
-﻿{

```

Comment:
@preeti for some reason the diff is messed up, but if you compare with a different tool you will notice the only change is from `=~ 'PAN-OS'` to `contains 'PAN-OS'`

Reply:
@erant10 - This will show up as incorrect history after merge. Can you please remove this file and redo the change for accurate diff? Thanks.

Reply:
That's what I did but for some reason it still does not show the correct diff...

Reply:
@erant10 - You might want to remove this file from the current PR - do a new PR with the file with just the changes. Suspecting if the file size and more files in this PR is causing the diff to be gnarly. Thanks.